google/resource_compute_image_generated_test.go

```diff
@@ -0,0 +1,80 @@
+// ----------------------------------------------------------------------------
+//
+//     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+//
+// ----------------------------------------------------------------------------
+//
+//     This file is automatically generated by Magic Modules and manual
+//     changes will be clobbered when the file is regenerated.
+//
+//     Please read more about how to change this file in
+//     .github/CONTRIBUTING.md.
+//
+// ----------------------------------------------------------------------------
+
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccComputeImage_imageBasicExample(t *testing.T) {
+	t.Parallel()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeImageDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeImage_imageBasicExample(acctest.RandString(10)),
+			},
+			{
+				ResourceName:            "google_compute_image.example",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"raw_disk"},
+			},
+		},
+	})
+}
+
+func testAccComputeImage_imageBasicExample(val string) string {
+	return fmt.Sprintf(`
+resource "google_compute_image" "example" {
+  name = ""
+
+  raw_disk {
+    source = "https://storage.googleapis.com/my-bucket/my-disk-image-tarball.tar.gz"
```

Comment:
Does this test pass? I assume this bucket is fake

Reply:
Ah right, I forgot about the examples creating tests.  Grabbed the tgz from the existing tests, working its way through now.